Automattic-Tracks-iOS/Event Logging/EventLoggingUploadManager.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
+import CocoaLumberjack
+import Sodium
+
+class EventLoggingUploadManager {
```

Comment:
Is this the main entry point for the WP/WC apps? If so, should this be `public`?

Reply:
I like the `enum` thing – and it's also suggested in the [RW style guide](https://github.com/raywenderlich/swift-style-guide/#constants).

Changed in 0c5b5bff1ba54eed15524251d91575c8ef89bb18.

Reply:
Just to reiterate https://github.com/Automattic/Automattic-Tracks-iOS/pull/127#discussion_r398788586, this still isn't part of the public API – that'll come in the next PR :)